seed/challenges/01-responsive-web-design/basic-html-and-html5.json

```diff
@@ -34,7 +34,7 @@
       "title": "Say Hello to HTML Elements",
       "description": [
         "Welcome to freeCodeCamp's coding challenges. These will walk you through web development step-by-step.",
-        "We recommend you do these in order, but feel free to skip around by clicking the map on the left.",
+        "We recommend you do these in order, but feel free to skip around by clicking the map in the navigation bar.",
```

Comment:
@jdeprey yes, you'll want to keep one commit as per our rules. The easiest may be to just amend this commit. So you can do the following.

```shell
... # Make changes
git add seed/challenges/01-responsive-web-design-basic-html-and-html5.json # The path will change depending on where you are
git commit --amend # This will just keep your previous commit on this branch.
git push origin fix/say-hello-map --force # You need the `--force` to overwrite the current commit
```

If you have questions about it, feel free to visit the [Contributors Help](https://gitter.im/FreeCodeCamp/HelpContributors) chat room if you have any questions about helping. We're there to help.